ui/ui-frontend/projects/pastis/src/app/profile/list-profile/load-profile/load-profile.component.ts

```diff
@@ -54,9 +58,15 @@ export interface LoadProfileConfig {
   selector: 'app-load-profile',
   templateUrl: './load-profile.component.html',
   styleUrl: './load-profile.component.scss',
-  imports: [FileSelectorComponent, PipesModule, MatDialogModule, VitamUILibraryModule],
+  imports: [FileSelectorComponent, PipesModule, MatDialogModule, VitamUILibraryModule, AsyncPipe, ReactiveFormsModule],
 })
 export class LoadProfileComponent {
+  protected files = new FormControl([]);
+  protected isEmpty$ = new BehaviorSubject(true).pipe(
+    switchMap(() => this.files.valueChanges.pipe(startWith(this.files.value))),
+    map((files: File[] | FileList) => files?.length === 0),
+  );
```

Comment:
On pourrait utiliser un signal
```suggestion
  protected isEmpty = toSignal(
    this.files.valueChanges.pipe(
      startWith(this.files.value),
      map((files: File[] | FileList) => files?.length === 0),
    ),
  );
```

Reply:
Ca doit être backporté vers 8.0